reg/tests.py

```diff
@@ -1,3 +1,8 @@
 from django.test import TestCase
+from django.urls import reverse
 
-# Create your tests here.
+class IndexTests(TestCase):
+
+  def test_get(self):
+    response = self.client.get(reverse(''))
```

Comment:
テストのとっかかりは良い感じで書けています。このコードでは、`reverse`のパラメータが指定されていないのでエラーになり、テスト失敗と表示されています。パラメータはurlsに書いてある`name`を使いますが、同じものを何回か使うので、`setUp`メソッドの中であらかじめ設定しておきます。ここでは、以下のように修正します（`[hoge]`のところを適切なものに変えてください）。
```suggestion
class IndexTests(TestCase):
  def setUp(self):
    self.url0 = [hoge]
    
  def test_get(self):
    response = self.client.get(self.url0)
```
一度テストコードを書いてしまえば、テストを動かすたびにこの場所をチェックしてくれます。ほかには、「はじめは空である」とか、「同じユーザー名では登録できない」などのテストをいれることを考えるとよいと思います。